app/controllers/api/v1/user_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::UserController < ApplicationController
+  def is_signed_in?
+    if user_signed_in?
+      render :json => {"signed_in" => true, "user" => current_user}.to_json()
+    else
+      render :json => {"signed_in" => false}.to_json()
```

Comment:
You shouldn't *need* to call `.to_json()` on these. Also, since I hate the hashrocket syntax, I would personally write them as: `render json: { signed_in: true, user: current_user }`.